x/delayedack/types/rollapp_packets_list_filter.go

```diff
@@ -26,6 +26,7 @@ func PendingByRollappIDByMaxHeight(
 				End:   commontypes.RollappPacketByStatusByRollappIDByProofHeightPrefix(rollappID, status, maxProofHeight+1), // inclusive end
 			},
 		},
+		FilterFunc: func(packet commontypes.RollappPacket) bool { return true },
```

Comment:
should define it (`func(packet commontypes.RollappPacket) bool { return true }` )and reuse for readability